src/doc/html.md

```diff
@@ -30,9 +30,20 @@ attribute to `<html>` as in this example:
 <html class="no-js" lang="en">
 ```
 
+
+## Document Title
```

Comment:
I think it's a good addition. Mostly people will know it but maybe forget about the search engine results.